read_repair_test.py

```diff
@@ -732,106 +730,6 @@ def stop_reads(*nodes, kind='all'):
     with _byteman_cycle(nodes, {'data': [data], 'digest': [digest], 'all': [data, digest]}[kind]):
         yield
 
-kcvv = lambda k, c, v1, v2: [k, c, v1, v2]
-
-
-class TestReadRepairGuarantees(Tester):
```

Comment:
@adelapena Just to confirm, this is removed entirely because it was added in 4.0 anyway?

Reply:
@maedhroz Yes, we initially did a literal migration of this class that we discarded because we considered that the tested cases were already covered by the existing tests. However, just removing this class feels a bit scary to me, perhaps we could keep it around by now with a deprecation note, WDYT?

Reply:
I don't see any reason to worry about removing it. If we did all this work over again (transient replication, etc.), we'd probably end up with exactly what we've done in this issue. On the other hand, it doesn't hurt to keep it around for 4.0. I'll defer to your and our second reviewer.

Reply:
@maedhroz as mentioned [on Slack|https://the-asf.slack.com/archives/CK23JSY2K/p1602201135179400], I have created [here](https://github.com/apache/cassandra-dtest/pull/95/commits/1d5c6460a9b5b744c29eacf1f7fddd2dde5ec4d0) a specific Pytest marker to skip tests ported to in-JVM only if we are not using vnodes.